.github/workflows/cms_ci_cd.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - name: Copy env variables to docker
         run: |
-          echo "${{ secrets.CMS_STAGE_ENV_FILE }}" > .env
+          echo "${{ secrets.CMS_STAGE_ENV_FILE }}" > cms/.env
```

Comment:
Ahh, good catch @agnessa !

Reply:
this didn't seem to help so I reverted 😅